files/galaxy/tpv/tools.yml

```
@@ -583,6 +583,8 @@ tools:
       - if: input_size >= 25
         fail: Too much data, please check if the input is correct.
 
+  inherits: toolshed.g2.bx.psu.edu/repos/nml/spades/spades/.*
```

You need to bring this line back...

```suggestion
  toolshed.g2.bx.psu.edu/repos/iuc/shovill/shovill/.*:
    inherits: toolshed.g2.bx.psu.edu/repos/nml/spades/spades/.*
```